pytx/scripts/get_data.py

```diff
@@ -0,0 +1,264 @@
+        args.output = 'threat_indicators.csv'
+        query(args)
+
+    else:
+
+        query(args)
+
+
+def query(args):
+
+    """
+    Query the ThreatExchange API at the specified endpoint.
+    """
+
+    # maximum number of indicators to fetch
+    result_limit = 1000
+
+    # write results to this stream
+    output_stream = '/dev/stdout' if not args.output else args.output
+
+    for day in range(args.days_back):
+
+        # format date parameters for HTTP request
+        until, until_str, since, since_str = utils.get_time_params(args.end_date, day, '%d-%m-%Y')
+
+        with open(output_stream, 'wb') as ostream:
+
+            print('Writing to %s...' % output_stream)
+
+            writer = csv.writer(ostream)
+
+            if args.object == 'exchange_member':
+
+                engine = ThreatExchangeMember
+
+                fields = [XM.ID, XM.NAME]
+
+                parameters = dict()
+
+            elif args.object == 'malware_analysis':
+
+                engine = Malware
+
+                fields = [
+                    MA.ID,
+                    MA.ADDED_ON,
+                    MA.CRX,
+                    MA.IMPHASH,
+                    MA.MD5,
+                    MA.PASSWORD,
+                    MA.PE_RICH_HEADER,
+                    MA.SAMPLE_TYPE,
+                    MA.SAMPLE_SIZE_COMPRESSED,
+                    MA.SHA1,
+                    MA.SHA256,
+                    MA.SHARE_LEVEL,
+                    MA.SSDEEP,
+                    MA.STATUS,
+                    MA.VICTIM_COUNT,
+                    MA.XPI,
+                ]
+
+                param_fields = Malware._default_fields
+                if args.full_sample:
+                    param_fields += ['sample_size', 'sample']
+
+                parameters = dict(
+                    fields=param_fields,
+                    limit=result_limit,
+                    text=args.text,
+                    strict_text=args.strict_text,
+                    sample_type=args.malware_type,
+                    status=args.status,
+                    share_level=args.share_level,
+                    since=since_str,
+                    until=until_str
+                )
+
+            elif args.object == 'malware_family':
+
+                engine = MalwareFamily
+
+                fields = [
+                    MF.ID,
+                    MF.ADDED_ON,
+                    MF.ALIASES,
+                    MF.DESCRIPTION,
+                    MF.FAMILY_TYPE,
+                    MF.MALICIOUS,
+                    MF.NAME,
+                    MF.SAMPLE_COUNT
+                ]
+
+                parameters = dict(
+                    fields=MalwareFamily._fields,
+                    limit=result_limit,
+                    text=args.text,
+                    strict_text=args.strict_text,
+                    since=since_str,
+                    until=until_str
+                )
+
+            elif args.object == 'threat_description':
```

Comment:
Should we change this to "threat_descriptor" so it aligns with how the other object types are used?